python/pyspark/mllib/linalg.py

```diff
@@ -633,6 +703,38 @@ def stringify(vector):
         """
         return str(vector)
 
+    @staticmethod
+    def squared_distance(a, b):
+        """
+        Squared distance between two vectors.
+        a and b can be of type SparseVector, DenseVector, np.ndarray
+        or array.array.
+
+        >>> a = Vectors.sparse(4, [(0, 1), (3, 4)])
+        >>> b = Vectors.dense([2, 5, 4, 1])
+        >>> a.squared_distance(b)
+        51.0
+        """
+        a, b = _convert_to_vector(a), _convert_to_vector(b)
+        return a.squared_distance(b)
+
+    @staticmethod
+    def norm(vec, p):
+        """
+        Find norm of the given vector.
+        """
+        return _convert_to_vector(vec).norm(p)
+
+    @staticmethod
+    def parse(vectorString):
```

Comment:
Can you please add doc, plus little doc tests?

Also, can the argument please be "s" to match Scala?